content/docs/2.13/authentication-providers/aws-secret-manager.md

```diff
@@ -0,0 +1,48 @@
++++ 
+title = "AWS Secret Manager" 
++++
+
+You can integrate AWS Secret Manager secrets into your trigger by configuring the `awsSecretManager` key in your KEDA scaling specification.
+
+The `podIdentity` section configures the usage of AWS pod identity with the provider set to AWS.
+
+The `credentials` section specifies AWS credentials, including the `accessKey` and `secretAccessKey`.
+
+- **accessKey:** Configuration for the AWS access key.
+- **secretAccessKey:** Configuration for the AWS secret access key.
+
+The `region` parameter is optional and represents the AWS region where the secret resides, defaulting to the default region if not specified.
+
+The `secrets` list within `awsSecretManager` defines the mapping between the AWS Secret Manager secret and the authentication parameter used in your application, including the parameter name, AWS Secret Manager secret name, and an optional version parameter, defaulting to the latest version if unspecified.
+
+### Prerequisites
+
+1. **AWS Credentials:**
+   - Ensure that AWS credentials with sufficient permissions are available. This can be achieved by setting the `TF_AWS_ACCESS_KEY` and `TF_AWS_SECRET_KEY` environment variables.
+
+2. **AWS Secret Manager Secret:**
+   - Create a secret in AWS Secret Manager containing the necessary sensitive information, such as database connection strings.
```

Comment:
I think that this section isn't necessary as it's something obvious IMHO. (and the point 1 it's not correct, that's only for our testing env)

Reply:
Alright I will remove it.